doc/cephadm/host-management.rst

```diff
@@ -183,6 +194,12 @@ The following host labels have a special meaning to cephadm.  All start with ``_
   an existing host that already contains Ceph daemons, it will cause cephadm to move
   those daemons elsewhere (except OSDs, which are not removed automatically).
 
+* ``_no_conf_keyring``: *Do not deploy config files or keyrings on this host*.
+
+  This label is effectively the same as ``_no_schedule`` but instead of working for
+  daemons it works for client keyrings and ceph conf files that are being managed
+  by cephadm
+
```

Comment:
here I'm a little bit lost/confused: the `_no_conf_keyring` label is supposed to keep the configuration but as of this statment it seems to prevent deploying config on the host! .. I think we have to reword this to better reflect what's the purpose of this label.

Reply:
`_no_conf_keyring` does in fact prevent cephadm from deploying the conf and keyring files on the host. I think the confusion is with the fact that the `--leave-conf-keyring` flag (or `--keep-conf-keyring` if I rename it as you suggested in another comment) tells the drain host function NOT to put this label on the host. To summarize:

- drain with NO `--leave-conf-keyring` flag -> Add `_no_conf_keyring` label and conf and keyring files get removed
- drain WITH `--leave-conf-keyring` flag -> do NOT add `_no_conf_keyring` label and the conf and keyring files are left as is.
- In both cases, add `_no_schedule` label that will cause daemons on the host to be removed

So in that sense the label does work exactly like `_no_schedule` but for conf and keyrings instead of daemons. It's just that we have an optional flag for the drain host function that tells us not to add the label as part of the drain process.

Reply:
Ok, I see thanks for the detailed explanation. Maybe the confusion in fact comes from the double negation usage but if you think about it in terms of how _no_schedule works then it's becomes easiser to understand.